recipes/gstreamer/all/conandata.yml

```diff
@@ -17,3 +17,18 @@ sources:
   "1.19.2":
     url: "https://gitlab.freedesktop.org/gstreamer/gstreamer/-/archive/1.19.2/gstreamer-1.19.2.tar.gz"
     sha256: "6ec8494867d9b58f145c0c357da631faf21c2c72c7cd8b637b440188fe904ab8"
+  "1.20.0":
```

Comment:
@mjimenofluendo Do you really need all those versions? Please, keep only what you need. More versions require more resources in terms of CI. Of someone need a patch version, a new PR can be opened later.

Reply:
I have a question, if I leave all the versions in the conandata.yml file and only leave 1.20.4 in the config.yml file, would I also spend CI resources?
I would like to give the possibility of at least being able to compile the version if necessary.

Reply:
The CI will not accept. Both conandata.yml and coonfig.yml need to be aligned, otherwise, the CI will detect it as an error and block your PR.

Reply:
I understand, we will leave the latest version then.